### PR TITLE
Prf lsh svd

### PR DESCRIPTION
**Pull request by Kunal30**

### CODE REVIEW

**Comment by AdityaGovardhan** [comment]
Rebase!

File: src/backend/utils.py [hunk 1]
```diff
@@ -6,11 +6,11 @@
 import matplotlib.pyplot as plt
 import matplotlib
 
-from backend.database_connection import DatabaseConnection
+from database_connection import DatabaseConnection
```
Comment (AdityaGovardhan) [inline src/backend/utils.py:9]: @vibnimit

File: src/backend/utils.py [hunk 2]
```diff
@@ -51,6 +51,8 @@ def get_dot_distance(vector1, vector2):
 def get_cosine_similarity(vector1, vector2):
     return spatial.distance.cosine(vector1, vector2)
 
+def get_euclidian_distance(vector1, vector2):
```
Comment (AdityaGovardhan) [inline src/backend/utils.py:51]: this function is already present above at line 30

Comment (Kunal30) [reply src/backend/utils.py:51]: dafaq mere waale me it was missing.... had to write it again..... looks like ur changes came

**Comment by AdityaGovardhan** [comment]
Added comments. Looks good to integrate. Will approve after @skuma149 and @vibnimit approval

**Review by skuma149 — DISMISSED** [review]
Sorting K images is being called from k similar images

**Review by AdityaGovardhan — APPROVED** [review]
Looks good to merge

File: src/backend/relevance_feedback.py [hunk 3]
```diff
@@ -67,15 +73,94 @@ def get_user_feedback(self,init_rank_list,q_name,caller='misc'):
...
+		After computing q_new, now we will train an SVM classifier on the basis of rel_items and irl_items
+		After training the SVM classifier we test it on a superset of data from LSH (possibly 1000+t)
+		We create an obj_feature_matrix on the basis of predicted relevant items from the classifier
+		We pass this to the get_most_m_similar_images() function
 
-	def get_Dec_Tree_based_feedback(self):
-		pass
+		Potenial Lines of code from this point onwards:
+		svm=SupportVectorMachine()
+		lsh=LSH()
```
Comment (skuma149) [inline src/backend/relevance_feedback.py:131]: Need to provide k and l value while initializing

**Comment by Kunal30** [comment]
These are just pseudo code comments don’t worry

**Review by skuma149 — APPROVED** [review]

**Pull request merged**